src/components/Login/Login.js

```diff
@@ -92,7 +92,8 @@ const Component = (props) => {
     if ((response || {}).status === 200) {
       const rawJwt = response.data;
       manageUserTokenWithLoader(rawJwt, provider, () => {
-        setIsHandlingToken(false);
+        // Note: no need to use setIsHandlingToken(false);
```

Comment:
I had warnings because I set a state on an unmounted component. So, I corrected that while at it.